CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.8.5] - 2022-02-15
+- Bugfix: memory leak with PostProcessing if no PP layer is present on the camera
```

Comment:
Bugfix: A memory leak no longer occurs with PostProcessing if no PP layer is present on the camera.